cmd/service/metro/metro.go

```diff
@@ -61,6 +62,11 @@ func Init(ctx context.Context, env string, componentName string) {
 		log.Fatal("admin credentials missing")
 	}
 
+	if appConfig.Encryption.Key == "" {
+		log.Fatal("encryption key missing")
+	}
+	encryption.RegisterEncryptionKey(appConfig.Encryption.Key)
```

Comment:
should we create a singlton instance and pass the whole encryption config than just key